packages/ui-default/templates/manage_user_priv.html

```diff
@@ -0,0 +1,100 @@
+{% extends "manage_base.html" %}
+{% block manage_content %}
+<div style="display: none" class="dialog__body--select-user">
+  <div>
+    <div class="row"><div class="columns">
+      <h1>{{ _('Select User') }}</h1>
+    </div></div>
+    <div class="row">
+      <div class="medium-6 columns">
+        <label>
+          {{ _('Username / UID') }}
+          <input name="user" type="text" class="textbox" autocomplete="off" data-autofocus>
+        </label>
+      </div>
+    </div>
+  </div>
+</div>
+<div style="display: none" class="dialog__body--set-priv">
+  <div>
+    <div class="row"><div class="columns">
+      <h1>{{ _('Set Priv') }}</h1>
+    </div></div>
+    <div class="row">
+      <div class="medium-12 columns">
+        <table class="data-table priv_table">
+          <colgroup>
+            <col class="col--description">
+            <col class="col--p">
+          </colgroup>
+          <thead>
+            <tr>
+              <th class="col--description">{{ _('Name') }}</th>
+              <th class="col--p">{{ _('Priv') }}</th>
+            </tr>
+          </thead>
+          <tbody>
+            {%- for name, priv in PRIV -%}
+            {%- if priv !== -1 -%}
+            <tr>
+              <td class="col--description">{{ _(name) }}</td>
+              <td class="col--p">
+                  <label class="compact checkbox">
+                    <input class="priv" name="{{ name }}" data-checkbox-group="priv" data-checkbox-range value="{{ priv }}" type="checkbox">
+                  </label>
+                </td>
+            </tr>
+            {%- endif -%}
+            {%- endfor -%}
+          </tbody>
+        </table>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="section">
+  <div class="section__header">
+    <h1 class="section__title">{{ _('User Priv') }}</h1>
+    <div class="section__tools">
+      <button class="primary rounded button" name="select_user"><span class="icon icon-search"></span>{{ _('Select User') }}</button>
+    </div>
+  </div>
+  {{ noscript_note.render() }}
+  <div class="section__body no-padding">
+    <table class="data-table">
+      <colgroup>
+        <col class="col--uid">
+        <col class="col--user">
+        <col class="col--priv">
+        <col class="col--actions">
+      </colgroup>
+      <thead>
+        <tr>
+          <th class="col--uid">{{ _('User ID') }}</th>
+          <th class="col--user">{{ _('Username') }}</th>
+          <th class="col--priv">{{ _('Priv') }}</th>
+          <th class="col--actions">{{ _('Actions') }}</th>
+        </tr>
+      </thead>
+      <tbody>
+        {%- for udoc in udocs -%}
```

Comment:
add another line `default` which controls `default.priv` in system setting

Reply:
Also this setting should change all user priv that equals to previous default priv